### PR TITLE
Add bias parameter to ConvNormActivation

### PR DESCRIPTION
**Pull request by NicolasHug**
This PR adds a `bias` parameter to `ConvNormActivation`. The current behaviour in `main` is to add a bias if there's no normalization layer, but in some cases like in RAFT, we need the biases even though there *is* a norm layer. For details, see:  https://github.com/NicolasHug/vision/blob/5f58ec164f6cce45c155815ab5f845d4773fad94/torchvision/models/video/_raft/raft.py#L15

### CODE REVIEW

**Review by datumbox — APPROVED** [review]
LGTM, just a minor typo and a missing `optional` on the documentation.

File: torchvision/ops/misc.py [hunk 1]
```diff
@@ -131,9 +132,12 @@ def __init__(
         activation_layer: Optional[Callable[..., torch.nn.Module]] = torch.nn.ReLU,
         dilation: int = 1,
         inplace: bool = True,
+        bias: Optional[bool] = None,
```
Comment (prabhat00155) [inline torchvision/ops/misc.py:135]: bias is a bool, shouldn't the default be False?

File: torchvision/ops/misc.py [hunk 2]
```diff
@@ -131,9 +132,12 @@ def __init__(
...
     ) -> None:
         if padding is None:
             padding = (kernel_size - 1) // 2 * dilation
+        if bias is None:
```
Comment (prabhat00155) [inline torchvision/ops/misc.py:139]: These two statements could be removed by adding
```
bias=bias or norm_layer is None
```
on line 150(given bias is bool and defaults to False).

**Review by prabhat00155 — COMMENTED** [review]
Thanks @NicolasHug, I have a couple of comments.

File: torchvision/ops/misc.py (hunk 1 above)
Comment (NicolasHug) [reply torchvision/ops/misc.py:135]: We want to keep BC so we can't have None be the default. Previously, bias was set to `norm_layer is not None`.

File: torchvision/ops/misc.py (hunk 2 above)
Comment (NicolasHug) [reply torchvision/ops/misc.py:139]: I don't think this is equivalent: This would override bias whenever bias is False. We only want to override bias when it's None.

Comment (prabhat00155) [reply torchvision/ops/misc.py:139]: I see, my understanding was wrong. I thought if bias is False we default back to `norm_layer is None`
